src/fdrs_gateway_serial.h

```diff
@@ -48,19 +52,48 @@ void getSerial() {
     }
     ln = s;
     newData = event_serial;
-    DBG("Incoming Serial.");
-
+    DBG("Incoming Serial: DR");
+      String data;
+      serializeJson(doc, data);
+      DBG("Serial data: " + data);
```

Comment:
JSON data output to the console should be displayed regardless of FDRS_DEBUG. Among other things, it's documented or mentioned somewhere to work that way, so we shouldn't change it.

Reply:
Timm, I'm not sure I understand that comment.  Can you elaborate a bit on:
1." JSON data output to the console should be displayed regardless of FDRS_DEBUG."
2. The idea is that the console interface always doubles as a data interface.

Are you saying that even if the DBG is not there we should see the data on the console?

Reply:
Yes. Even if `FDRS_DEBUG` is disabled, the "Serial" (as well as "Serial1") interface should print out DataReadings when `sendSerial()` is called. I also think it best if they are delivered surrounded by line-breaks. Just in case a front end application is ingesting from that interface and needs it more standardized. Does that help?

It was originally implemented due to the ESP8266 having only one serial interface. If you need/want to re-use that interface for GPS or something, maybe we could have a way to explicitly disable this. Afterthought: A way to fully disable the serial interfaces might be called-for if we need that.

Side note: I might disable the new indentations in the debug console when at level 0.  The blank space is just a bit jarring (probably just to me) but it's also easier to read when info is closer to the timestamp.